examples/tomo/astra_parallel_2d_fp.py

```diff
@@ -1,67 +0,0 @@
-# Copyright 2014, 2015 The ODL development group
```

Comment:
Trivial to split the I suppose. Didnt want to many almost identical files, but it may be worth it I suppose.